apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -1608,6 +1628,8 @@ private void createTraceAndAssert(Trace trace, String apiKey, String workspaceNa
     }
 
     private UUID createAndAssert(Experiment expectedExperiment, String apiKey, String workspaceName) {
+        Mockito.reset(defaultEventBus);
```

Comment:
This is due to the real event bus being replaced by an instance of mockito, and I reset it to avoid one test impacting the other. Otherwise I would need create a new application context for each test.